### PR TITLE
jars command

### PR DESCRIPTION
**Pull request by eed3si9n**
This adds a Bazel query wrapper called `jars` command.
While debugging the bazel-multiversion resolution, I often want to find
out what the JAR file(s) were selected given some module like JUnit or
Netty.

### Example

```
multiversion jars io.netty:netty-common
```

This will print out:

```
@maven//:io.netty/netty-common/4.1.59.Final.jar
```

### CODE REVIEW

File: multiversion/src/main/scala/multiversion/commands/QueryBasedCommand.scala [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  def bazel: Path
+  def bazelArgs: List[String]
+  def runCustomQuery(): Result[List[String]]
+
+  override def run(): Int = app.complete(runResult())
+
+  def runResult(): Result[Unit] =
+    for {
+      r <- runCustomQuery()
+    } yield r.foreach(Console.out.println(_))
```
Comment (Duhemm) [inline multiversion/src/main/scala/multiversion/commands/QueryBasedCommand.scala:26]:
```suggestion
    } yield r.foreach(app.println(_))
```

I think moped swaps the streams in tests.

File: multiversion/src/main/scala/multiversion/commands/QueryBasedCommand.scala [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    val lines =
+      app
+        .process((command ::: bazelArgs).map(x => (x: Shellable)): _*)
+        .call()
+        .out
+        .text()
+        .linesIterator
+        .toList
+    ValueResult(lines)
+  }
```
Comment (Duhemm) [inline multiversion/src/main/scala/multiversion/commands/QueryBasedCommand.scala:50]: There exists `BazelUtil.bazel` to run Bazel commands and display a progress bar for long running commands:
https://github.com/twitter/bazel-multiversion/blob/c09296ee8d36922fe088b909937b65effebcfabb/multiversion/src/main/scala/multiversion/BazelUtil.scala#L30-L35

There's also a helper to run `bazel query` in `LintCommand`:
https://github.com/twitter/bazel-multiversion/blob/c09296ee8d36922fe088b909937b65effebcfabb/multiversion/src/main/scala/multiversion/commands/LintCommand.scala#L100-L111

What do you think of moving `runQuery` to `BazelUtil` and reusing it?

Comment (eed3si9n) [reply multiversion/src/main/scala/multiversion/commands/QueryBasedCommand.scala:50]: `runQuery` captures the output as Protobuf, but that doesn't work for `--output graph`, so I'm not using it.

In addition, `BazelUtil.bazel` adds a progress bar, which I didn't want for query wrappers. We can still move `runQueryStr` to `BazelUtil`.

**Comment by eed3si9n** [comment]
I'll merge this and rebase the next PR on top

**Pull request merged**